oci/private/image.bzl

```diff
@@ -96,6 +97,35 @@ def _platform_str(os, arch, variant = None):
         parts["variant"] = variant
     return json.encode(parts)
 
+def _calculate_descriptor(ctx, idx, layer, zstd, jq, coreutils, regctl):
+    descriptor = ctx.actions.declare_file("%s.%s.descriptor.json" % (ctx.label.name, idx))
+    args = ctx.actions.args()
+    args.add(layer)
+    args.add(descriptor)
+    ctx.actions.run(
```

Comment:
in theory we should be adding the windows batch wrapper around new shell script executables. I guess we can come back to it later in 2.0 release

Reply:
i can do that here, good catch.